src/Apps/Order/Components/BankAccountPicker.tsx

```diff
@@ -145,6 +145,7 @@ export const BankAccountPicker: FC<Props> = props => {
         <>
           {bankAccountHasInsufficientFunds && <InsufficientFundsError />}
           <Button
+            data-test="bank-transfer-save-existing"
```

Comment:
This is bit nitpicky, but in general: but why not `bankTransferSaveExisting`? We use camelCase everywhere else, and this string will still be used in code (tests)